map_test.go

```diff
@@ -21,16 +21,14 @@ import (
 	"github.com/go-quicktest/qt"
 )
 
-var (
```

Comment:
No arbitrary style changes, please.

Reply:
My apologies. Turns out my editor was using `gofumpt` instead of `gofmt` which caused changes like this to sneak in.